core/services/flux_monitor.go

```diff
@@ -155,31 +184,32 @@ func connectCheckers(ctx context.Context, jobMap map[string][]DeviationChecker,
 	}
 }
 
-func (fm *concreteFluxMonitor) addAction(ctx context.Context, connected bool, job *models.JobSpec, jobMap map[string][]DeviationChecker) error {
-	if _, ok := jobMap[job.ID.String()]; ok {
-		return fmt.Errorf("job %s has already been added to flux monitor", job.ID)
-	}
-	validCheckers := []DeviationChecker{}
-	for _, initr := range job.InitiatorsFor(models.InitiatorFluxMonitor) {
-		logger.Debugw("Adding job to flux monitor",
-			"job", job.ID.String(),
-			"initr", initr.ID,
+func (fm *concreteFluxMonitor) addAction(
+	ctx context.Context,
+	connected bool,
+	jobMap map[string][]DeviationChecker,
+	store *store.Store,
+	jobSpecID string,
+	checkers ...DeviationChecker,
```

Comment:
This splat operator is not being used.

Reply:
Lets agree to disagree about this, but changed this for the PR